rllib/algorithms/appo/tf/appo_tf_rl_module.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+
+from ray.rllib.algorithms.ppo.tf.ppo_tf_rl_module import PPOTfRLModule
+from ray.rllib.core.models.base import ACTOR
+from ray.rllib.core.models.tf.encoder import ENCODER_OUT
+from ray.rllib.core.rl_module.rl_module_with_target_networks import (
+    RLModuleWithTargetNetworks,
+)
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.nested_dict import NestedDict
+
+_, tf, _ = try_import_tf()
+
+OLD_ACTION_DIST_KEY = "old_action_dist"
+OLD_ACTION_DIST_LOGITS_KEY = "old_action_dist_logits"
+
+
+class APPOTfRLModule(PPOTfRLModule, RLModuleWithTargetNetworks):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
We should call explicitly both super constructors:

```
PPOTfRLModule.__init__(self, ..)
RLModuleWithTargetNetworks.__init__(self, ...)
```

Or the other way around.

Reply:
done